servers/rest-services/src/main/java/org/projectnessie/services/rest/RestV2TreeResource.java

```diff
@@ -431,7 +439,59 @@ public MergeResponse mergeRefIntoBranch(String branch, Merge merge)
   @Override
   public CommitResponse commitMultipleOperations(String branch, Operations operations)
       throws NessieNotFoundException, NessieConflictException {
+    ImmutableOperations.Builder ops =
+        ImmutableOperations.builder()
+            .from(operations)
+            .commitMeta(commitMeta(CommitMeta.builder().from(operations.getCommitMeta())).build());
+
     ParsedReference ref = parseRefPathString(branch);
-    return tree().commitMultipleOperations(ref.name(), ref.hashWithRelativeSpec(), operations);
+    return tree().commitMultipleOperations(ref.name(), ref.hashWithRelativeSpec(), ops.build());
+  }
+
+  CommitMeta.Builder commitMeta(CommitMeta.Builder commitMeta) {
+    httpHeaders
+        .getRequestHeaders()
+        .forEach(
+            (k, v) -> {
+              if (!v.isEmpty()) {
+                String lower = k.toLowerCase(Locale.ROOT);
+                switch (lower) {
+                  case "nessie-commit-message":
+                    v.stream()
+                        .map(String::trim)
+                        .filter(s -> !s.isEmpty())
+                        .findFirst()
+                        .ifPresent(commitMeta::message);
+                    break;
+                  case "nessie-commit-authors":
+                    v.stream()
+                        .flatMap(s -> Arrays.stream(s.split(",")))
```

Comment:
Sorry, if I caused confusion with my comma-separation comment :sweat_smile: I meant it for client-side config variables only. Headers already support multiple values, so it looks a bit odd to have multiple lists of comma-separated values here. I think we can rely on the standard HTTP way of processing multiple headers here, but split config strings only on the client side. WDYT?

Reply:
This is to have feature-parity when you configre that via iceberg

Reply:
(sigh)